docs/dev/setup.rst

```diff
@@ -257,6 +258,127 @@ You can stop the service using ``sudo service artemis stop`` and restart it usin
 
 Logs can be fetched using ``sudo journalctl -u artemis -f -n 200``.
 
+Run the server via Docker
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Artemis provides a docker image named ``ghcr.io/ls1intum/artemis:<VERSION>``.
+The current develop branch will be deployed as `latest` version.
+Releases like ``5.7.1`` are deployed as ``ghcr.io/ls1intum/artemis:5.7.1``.
+The easiest way to configure a local deployment via docker is a deployment with a docker-compose file.
+You could use a compose file similar to this (as an example this deployment uses the gitlab-jenkins configuration of artemis:
+
+::
+
+    version: '3'
+
+    services:
+      gitlab:
+        image: gitlab/gitlab-ce
+        restart: unless-stopped
+        volumes:
+          - ./data/gitlab/config:/etc/gitlab
+          - ./data/gitlab/logs:/var/log/gitlab
+          - $BACKUP_DIR/gitlab:/var/opt/gitlab/backups # Optional but useful
```

Comment:
```suggestion
          # - $BACKUP_DIR/gitlab:/var/opt/gitlab/backups # Optional but useful
```
Maybe it should be commented out per default as it is optional.

Alternatively, as the Jenkins backup dir is not marked as optional, this shouldn’t be either or both.

Reply:
Yes they should be optional both :)